lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -1,17 +1,24 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:get/get.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:jmap_dart_client/jmap/core/session/session.dart';
 import 'package:model/model.dart';
 import 'package:tmail_ui_user/features/base/base_controller.dart';
 import 'package:tmail_ui_user/features/mailbox/presentation/mailbox_controller.dart';
+import 'package:tmail_ui_user/features/thread/presentation/thread_controller.dart';
 
 class MailboxDashBoardController extends BaseController {
 
+  final mailboxController = Get.find<MailboxController>();
+  final threadController = Get.find<ThreadController>();
```

Comment:
Why not build as the model `subscriber`?. `MailboxDashBoardController` should not know about subscribers: `MailboxController`, `ThreadController` and other controller in the future. All the controllers should listen to the change of `MailboxDashBoardController`.
Fx: Session will be the Obs type -> `MailboxController` listen to that variable, and react to the change of that (`getAllMailboxAction()`).